lib/player.rb

```diff
@@ -1,11 +1,11 @@
 # frozen_string_literal: true
 
-module Player
+class Player
   def position_taken?(board, index)
     !(board[index].nil? || board[index] == ' ')
   end
 
   def valid_move?(board, index)
-    index.between?(0, 8) && !position_taken?(board, index)
+    index.between?(0, 9) && !position_taken?(board, index)
   end
 end
```

Comment:
The player class is doing very little and performing the wrong task also, player is an object, you want to think of what a player does in the tic-tac-toe game these are the things the player class should handle.

A player doesn't check if the position of a board is taken, this is the job of the game,  also a player does check if a move is valid this is the job of the game also.

Things are player does. 
Set name
pick a symbol
Make a move